src/Sulu/Bundle/AdminBundle/Resources/js/containers/Sidebar/types.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import type {ComponentType} from 'react';
+
+export type SidebarViewOptions = {};
+
+export type SidebarView = ComponentType<SidebarViewOptions>;
+
+export type Sizes = 'small' | 'medium' | 'large';
```

Comment:
Until now we have named types like this `Size`, because it is not an array but represents the type for a single size.